.github/workflows/enforce-issue-template.yml

```diff
@@ -0,0 +1,35 @@
+name: enforce issue template
+on:
+  issues:
+    types: ['opened']
```

Comment:
I'd say include `'updated'` too to avoid cheaters?

Reply:
Meh, cheaters gonna cheat, and I don't think we should try too hard to prevent it. Instead, I think we should [ban](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/blocking-a-user-from-your-organization) cheaters (24h, with notification).

Reply:
Cheater ban policy would theoretically be documented on [open.sentry.io](https://github.com/getsentry/open-source-meta/issues/6) eventually.